packages/components/src/components/SquigglePlayground/index.tsx

```diff
@@ -187,15 +188,27 @@ export const SquigglePlayground: React.FC<PlaygroundProps> = (props) => {
   const leftPanelRef = useRef<HTMLDivElement | null>(null);
 
   const withEditor = (
-    <div className="flex mt-2">
+    <div className="mt-2 flex-row flex">
+      <ResizableBox
+        width={400}
```

Comment:
Is it possible to make this 50%, like it was before?

Reply:
Not trivial with react-resizable. My previous solution was overtaking existing containers, without pre-initializing width, but this is not the case now :(

I can some onResize atrocity to initialize it to 50% by default, but it won't look pretty.